index.html

```diff
@@ -9,6 +9,55 @@
     <script src='app.js'></script>
 </head>
 <body>
-   
+	<p class="slider slider-edo">edo</p>
+
+	<div class="map-container">
+	  <img src="nigeria-map.jpg" alt="Map of Nigeria" />
+	</div>
+
+
+	<script src="https://unpkg.com/interactjs/dist/interact.min.js"></script>
+
+
+
+	<!-- 
+		*
+		*
+			Programming Script
+		*
+		*
+	-->
+	<script>

```

Comment:
We may need to move all the code on the script to the app.js file just to have that separation of cocern

Reply:
work in progress